tests/test_assessment.py

```diff
@@ -9030,6 +9030,60 @@ def test_can_submit_foreign_language_text_response(self):
         self.assertTrue(data['correct'])
         self.assertIn('<p/>', data['feedback'])
 
+    def test_can_edit_max_strings(self):
+        item = self.create_item()
+        old_max_strings = item['question']['maxStrings']
```

Comment:
Do I read this correctly that the test depends on ``create_item()`` to set the value of ``maxStrings`` to something other than 55?